[Flaky CI]: test_outcome.py::test_global_rate_limit_by_namespace

### Flakiness Type

Assertion failure

### Name of Test

tests/integration/test_outcome.py::test_global_rate_limit_by_namespace

### Link to Test Run

https://github.com/getsentry/relay/actions/runs/8065335723/job/22030936697?pr=3168

### Details

Got the following assertion failure:

```
=================================== FAILURES ===================================
_____________________ test_global_rate_limit_by_namespace ______________________
[gw0] linux -- Python 3.10.13 /home/runner/work/relay/relay/.venv/bin/python3
Traceback (most recent call last):
  File "/home/runner/work/relay/relay/tests/integration/test_outcome.py", line 2042, in test_global_rate_limit_by_namespace
    assert len(outcomes) == 1
AssertionError: assert 0 == 1
 +  where 0 = len([])
```

--- a/tests/integration/fixtures/processing.py
+++ b/tests/integration/fixtures/processing.py
@@ -156,7 +156,7 @@ class ConsumerBase:
         self.consumer = consumer
         self.test_producer = kafka_producer(options)
         self.topic_name = topic_name
-        self.timeout = timeout or 1
+        self.timeout = timeout or 2
 
         # Connect to the topic and poll a first test message.
         # First poll takes forever, the next ones are fast.

